### DISCUSSION TITLE
How to save image or video in an album [IOS]

### DISCUSSION THREAD

**Discussion by tvqphong001** [fluttercandies/flutter_photo_manager#946] [Q&A] [answered]
## Expect
in Android I can save relativePath like 'Pictures/MyAlbum/' and load Images or Videos I saved in AssetPathEntity with name is MyAlbum but it not in IOS
```
final entity = await PhotoManager.editor.saveVideo(
        file,
        relativePath: Platform.isAndroid ? 'Pictures/MyAlbum/' : 'MyAlbum/,
        title: file.path.split('/').last, 
      );
```

I see you have function PhotoManager.editor.copyAssetToPath have copy to AssetPathEntity, can you suppor save Image or Video use AssetPathEntity
if you can support save use AssetPathEntity I can save to any albums I want

**Comment by AlexV525** [comment]
This is not supported at present, `relativePath` is only used for Android 29+.

**Comment by CaiJingLong** [comment, marked as answer]
The photo albums of iOS and Android are abstract concepts in photo_manager, and they are not exactly the same.
The iOS Asset is unique, and adding it to the Album is like adding a "shortcut" to the target album.
In Android, an entity file corresponds to an Asset, and similarly, an album corresponds to a folder, so we can save an asset to album, because this is save file to folder. 

So, back to the question, you cannot save an asset to the iOS photo album. If you want to do this, you can only save it to 'Recent' and copy a "shortcut" of the newly added asset to the photo album.
